Olha/index.js

```diff
@@ -0,0 +1,99 @@
+// 1. Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+let integer1 = Math.floor(Math.random() * 150);
+console.log("integer1:", integer1);
+let integer2 = Math.floor(Math.random() * 150);
+console.log("integer2:", integer2);
+
+let withinTheRange = (integer1 >= 50 && integer1 <= 99) && (integer2 >= 50 && integer2 <= 99);
```

Comment:
In this task we have to check if either of them are in the range not both of them!

Reply:
Thanks a million for checking this, Ciamac!
I should indeed be more careful when I read instructions (^-^)
Fixed.

Reply:
Any time :)